protos/in_toto_attestation/predicates/vuln/v1/vuln.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package in_toto_attestation.predicates.vuln.v1;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/in-toto/attestation/go/predicates/vuln/v1";
+option java_package = "io.github.intoto.attestation.predicates.vuln.v1";
+
+// Validation of all fields is left to the users of this proto.
+message Vuln {
+  Scanner scanner = 1;
+  ScanMetadata scan_metadata = 2;
+}
+
+message Scanner {
+  string uri = 1;
+  string version = 2;
+  VulnDatabase database = 3;
+  Result result = 4;
```

Comment:
If I'm reading this correctly I think `result` is supposed to be a list, but `vulnerability` is supposed to be singular.

https://github.com/in-toto/attestation/blob/main/spec/predicates/vuln.md#:~:text=scanner.result%2C%20required-,list,-The%20result%20contains

Reply:
I am thinking the spec shouldn't consider `result` as a list. It looks logical to me to have results as an object where there is a list of vulnerabilities. What do you think ?

Reply:
So I am proposing to change the spec so the result is singular and the vulnerability is a list.

Reply:
I think that's fine with me (especially if we're updating to v0.2).  if that's what you want can you update the docs in https://github.com/in-toto/attestation/blob/main/spec/predicates/vuln.md  and the example there too?